check-tree-sitter-crates.sh

```diff
@@ -0,0 +1,107 @@
+#!/bin/bash
+
+# Stop at the first error
+set -e
+
+# Get all tree-sitter crates from the analyzed branch Cargo.toml
+TS_CRATES=`grep "tree-sitter-*" Cargo.toml | tr -d ' '`
+
+# Disable/Enable CI flag
+RUN_CI="no"
+
+# Temporary master branch Cargo.toml filename
+MASTER_CARGO_TOML="master-cargo.toml"
+
+# Download master branch Cargo.toml and save it in a temporary file
+wget -LqO - https://raw.githubusercontent.com/mozilla/rust-code-analysis/master/Cargo.toml | tr -d ' ' > $MASTER_CARGO_TOML
```

Comment:
We could use git to get the content of the Cargo.toml file before the PR.

Reply:
Note: this won't work if we start using branches (as we'd always compare to master, we might end up running the checks always in the branch)